lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
+  attr_reader :category, :length, :width
 
   def initialize(category, length, width)
     @category = category
-    @length = 10
-    @width = width
+    @length = length.to_i
+    @width = width.to_i
+    @status = false
```

Comment:
we'd probably call this something like `@is_painted`, since `@status` could be the status of a great many things...